index.js

```diff
@@ -60,19 +60,40 @@ const Resizer = ({ array, updateCallback }) => {
   `;
 };
 
+let mouseDown = false;
+let dragType = 1;
+
 const Input = ({ array, updateCallback }) => {
-  const updateCell = (rowIndex, colIndex) => () => {
+  const updateCell = (rowIndex, colIndex) => {
     array[rowIndex][colIndex] ^= 1;
     updateCallback(array);
   };
 
+  const updateCellDrag = (rowIndex, colIndex) => {
+    array[rowIndex][colIndex] = dragType;
+    updateCallback(array);
+  };
+
   const createRow = (row, rowIndex) => html`
     <div>
       ${row.map(
         (active, colIndex) => html`
           <button
             class=${active ? "active cell" : "cell"}
-            onClick=${updateCell(rowIndex, colIndex)}
+            onClick=${() => {updateCell(rowIndex, colIndex)}}
```

Comment:
If we're already changing the cell value in onMouseDown and in onMouseOver, we definitely don't need to be changing it onClick too!

In fact, I just had a quick test, and all that is needed is:
```js
onMouseOver=${(e) => {
    if (e.buttons) updateCell(rowIndex, colIndex);
}}
```